src/pages/home/report/ReportActionCompose.js

```diff
@@ -324,6 +358,69 @@ class ReportActionCompose extends React.Component {
         this.setState({maxLines});
     }
 
+    /**
+     * Calculates and cares about the content of an Emoji Suggester
+     */
+    calculateEmojiSuggestion() {
+        const leftString = this.state.value.substring(0, this.state.selection.end);
+        const colonIndex = leftString.lastIndexOf(':');
+        const isCurrentlyShowingEmojiSuggestion = this.isLikeEmojiCode(this.state.value, this.state.selection.end);
+
+        // the larger composerHeight the less space for EmojiPicker, Pixel 2 has pretty small screen and this value equal 5.3
+        const isEnoughSpaceForSuggestion = this.props.windowHeight / this.state.composerHeight >= 6.8;
```

Comment:
I am a little concerned about hardcoding the height values like this for devices. I am going to take feedback from @stitesExpensify / @roryabraham here.

Reply:
If we at all keep the value, then we should move to CONST so that we've got some context when we read the value.

Reply:
Yeah it does feel a bit weird. So to clarify, this line is checking how much of the entire screen is taken up by the window height? And if it is smaller than a certain ratio then we need to reduce the size of the picker, but it doesn't reduce by a static height we just remove a row? Is that right?

Reply:
@perunt did you get a chance to look at this comment?

Reply:
That's not an excellent example to follow in CONST. The flow is confusing. It goes like this: 'isSmallScreenWidth - if the screen is narrow, it is potentially a smartphone.' It compares the screen's height and the input's height to decide whether there is enough space for a large suggestion. 
We can implement some handlers to handle dimensions better, especially when maintaining an extensive screen list.